.github/workflows/gui.yml

```diff
@@ -506,12 +506,6 @@ jobs:
           VITE_ENSO_MAPBOX_API_TOKEN: ${{ vars.ENSO_MAPBOX_API_TOKEN }}
           WIN_CSC_KEY_PASSWORD: ${{ secrets.MICROSOFT_CODE_SIGNING_CERT_PASSWORD }}
           WIN_CSC_LINK: ${{ secrets.MICROSOFT_CODE_SIGNING_CERT }}
-      - run: corepack pnpm -r --filter enso exec playwright test
```

Comment:
This file is labeled with

```yaml
 # This file is auto-generated. Do not edit it manually!
# Edit the enso_build::ci_gen module instead and run `cargo run --package enso-build-ci-gen`.
```

that's why I believe it is a **good idea** to remove this section manually. Next time someone tries to run `cargo run --package enso-build-ci-gen` this section will be regenerated back.

However now, the [Windows CI instability](https://discord.com/channels/401396655599124480/1201853512409939968/1308688656306929714) is _"blocking everybody's progress"_, and the tests should be disabled asap and reasons for the failures  as well as possible fixes investigated in another PR.